src/globus_sdk/tokenstorage/v2/validating_token_storage/context.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+
+import dataclasses
+
+
+@dataclasses.dataclass
+class TokenValidationContext:
+    """
+    A data object of information available to
+    :class:`TokenDataValidators <TokenDataValidator>` during validation.
+
+    :ivar str | None prior_identity_id: The identity ID associated with the
```

Comment:
I want to thank you for introducing this `ivar` syntax. I think you've used it in another PR or two but I don't think I've seen it elsewhere, and it's something that I'll be using moving forward. Thanks! :heart: